terraform/mesh_aws/eventbridge_send_message.tf

```diff
@@ -0,0 +1,86 @@
+resource "aws_cloudwatch_event_rule" "send_message_event" {
+  name        = "${local.name}-send-messages"
+  description = "${local.name}-send-messages"
+  event_pattern = jsonencode({
+    source = [
+      "aws.s3"
+    ]
+    detail-type = [
+      "AWS API Call via CloudTrail"
+    ]
+    detail = {
+      eventSource = [
+        "s3.amazonaws.com"
+      ]
+      eventName = [
+        "PutObject",
+        "CompleteMultipartUpload"
+      ]
+      requestParameters = {
+        bucketName = [
+          aws_s3_bucket.mesh.id
+        ]
+        key = [
+          {
+            prefix = "outbound"
```

Comment:
is this correct? should it be;
`prefix = "outbound/*"`

Reply:
We want this trigger to fire for all configured outbound mailboxes, which look like:
"outbound_X0000/"
"outbound_X0001/"

We can't use slashes in the file path, as that would not work with the SSM Parameter splitting on "/" used to configure the lambdas

Reply:
does it need to be `prefix = "outbound*"` then?

Reply:
No because I believe it will try to interpret that as a literal `*` in a key, not a glob

Reply:
ah ok, I've learnt something then :)